### ISSUE TITLE
Custom Javascript?

### ISSUE THREAD

**Issue by jevdemon** [idno/Known#2977]
I just upgraded to the latest version of Known from a much older version. 
The new version no longer includes an option to add custom Javascript to my pages. My previous installation used AnchorJS to add anchors and links to my docs.  I simply added the additional Javascript using the custom Javascript menu option.
The new version of Known no longer has a custom Javascript menu option.

How would I go about setting up custom Javascript in the current version of Known?

I am currently on Known Version: 1.2.2+2020061101

Thanks!

John

**Comment by lindner** [comment]
Did this functionality move to a different plugin?  Seems weird that we lost this capability...

**Comment by jevdemon** [comment]
It looks like lost functionality. I've been through all of the community plugins and none of them appear to support custom JavaScript. The menu option in the configure site menu is completely gone. I'm not sure when it disappeared I was on a much older version of Known.
I just wasn't sure if this feature was completely deprecated and if it was, is there an alternate approach for adding custom JavaScript?
Thanks!

**Comment by mapkyca** [comment]
Is it this one you want? https://github.com/idno/FooterJS

**Comment by jevdemon** [comment]
Sorry for such a late reply.  I tried installing the FooterJS plugin per the instructions and it never shows up in the list of available plugins.  Here's how I did it: 

1. git clone https://github.com/idno/FooterJS.git in the IdnoPlugins folder of my Known installation.
2. I expect to see a FooterJS plugin option on my Known Admin Plugins page but the FooterJS plugin never shows up.

I'm using what seems to be the current version of Known (Version: 1.2.2+2020061101).  
I've seen this behavior with other plugins too, not just FooterJS.  Did plugin handling change with this or a recent version of Known? I've never had an issue installng plugins before. 

Thanks!
